pulsar-function-go/pf/instance.go

```diff
@@ -227,6 +231,7 @@ func (gi *GoInstance) processResult(msgInput pulsar.Message, output []byte) {
 					} else if effectivelyOnce {
 						panic("in effectively-once case, this is error.")
 					}
+					log.Fatal(e)
```

Comment:
does log.fatal() kill the process? if thats so then the logic here is wrong.

Reply:
Doesn’t panic kill the process for EO?

Reply:
> does log.fatal() kill the process? if thats so then the logic here is wrong.

log.fatal() just print a line of logs and won't stop the thread. The last parameter in sendasync() is an anonymous function. If it fails, the program will crash here.

Reply:
> Doesn’t panic kill the process for EO?

yes